### ISSUE TITLE
About getting the current content of a document

### ISSUE THREAD

**Issue by wf-soft** [R2D221/WebView2.DOM#2]
For example, when I click the button to run JS, I find that what I get from the document content is not the final document content, it is a document when some content is not fully loaded

**Comment by wf-soft** [comment]
Maybe it's because the content in ifram is empty. How can I load the ifram completely without opening a new window。

**Comment by R2D221** [comment]
Sorry, I haven't tested iframes yet.

When you use DOMContentLoaded directly in C#, it should fire when the document is fully loaded, but the resources not yet.

Maybe you could try with the NavigationCompleted event instead.

**Comment by wf-soft** [comment]
Yes, it seems that the contents of the iframe cannot be obtained。

**Comment by wf-soft** [comment]
Even if the page is fully loaded, I can't get it from window.document Get htmliframeelement content

**Comment by wf-soft** [comment]
Maybe this is a cross domain problem
